Exercise_A/index.html

```diff
@@ -1,16 +1,18 @@
-<!DOCTYPE html>
-<html class="no-js">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="" />
-  </head>
-  <body>
-    <!-- CODE HERE -->
-
-    <script src="" async defer></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title></title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="" />
+  </head>
+  <body>
+    <!-- CODE HERE -->
+    <a href="about.html">
+      <img src="img_girl.jpg" alt="Girl in a jacket" width="500" height="600">
+    </a>
+    <script src="" async defer></script>
+  </body>
+</html>

```

Comment:
Hello Nicole 
Great work with this, as i am able to click the picture and then it links to the next page with the email address link